### PR TITLE
Create scholarships.yml, bursaries.yml and begin population

### PR DESCRIPTION
**Pull request by MylesJarvis**
### Trello card

- https://trello.com/c/2WHavuTv

### CODE REVIEW

File: config/values/bursaries.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+bursaries:
+  SKE: £175
```
Comment (gemmadallmandfe) [inline config/values/bursaries.yml:2]:
```suggestion
  SKE-weekly: £175
```
worth giving more context?

File: config/values/bursaries.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  veteran:
+    full-amount: £40,000
+    installment-amount: £20,000
+## implemented above this line
+  undergraduate: £9,000
+  postgraduate:
```
Comment (gemmadallmandfe) [inline config/values/bursaries.yml:8]: could these be changed to be in alphabetical order? will make it much easier to update :-)

**Review by gemmadallmandfe — COMMENTED** [review]
@MylesJarvis A few suggestions - but happy for you to ignore!

File: config/values/bursaries.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    chemistry: £28,000
+    computing: £28,000
+    maths: £28,000
+    physics: £28,000
+    biology: £25,000
+    design-and-technology: £25,000
+    languages-french-german-spanish: £25,000
```
[the remark this answers is not in the document]
Comment (MylesJarvis) [reply config/values/bursaries.yml:15]: I'm going to disagree with this one!

**Pull request closed without merging**